LanguageFeatures/Subtyping/test_types/records_fail_A01.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion We say that a type T0 is a subtype of a type T1 (written
+/// T0 <: T1) when:
+/// A record type A is a subtype of record type B iff they have same shape and
+/// the types of all fields of A are subtypes of the corresponding field types
+/// of B
+///
+/// @description Check that if type T0 is a record with the other shape than T1
+/// then T0 is not subtype of T1
```

Comment:
How does this test establish that said subtype relation does not exist? Same question for the next 6 tests.